webpack.dev.config.js

```diff
@@ -1,27 +1,21 @@
 var path = require("path");
 var webpack = require("webpack");
+var projectRoot = path.resolve(__dirname, '../');
 
 var loaders = [
 	{
-		"test": /\.js?$/,
-		"exclude": /node_modules/,
-		"loader": "babel"
+		test: /\.vue$/,
+		loader: 'vue'
 	},
 	{
-		"test": /\.css?$/,
-		"loader": "style!css"
+		test: /\.js$/,
+		loader: 'babel',
+		include: projectRoot,
+		exclude: /node_modules/
 	},
 	{
-		"test": /\.scss?$/,
-		"loader": "style!css!sass"
-	},
-	{
-		"test": /\.jade?$/,
-		"loader": "jade"
-	},
-	{
```

Comment:
Well, we don't require any `.jade`, `.scss` or `.css` files. They were never necessary (vue-loader take care of this).
It is also part of my temporary debug changes.
It can came back we everything is working again.